templates/studygroups/signup.html

```diff
@@ -5,11 +5,27 @@
 {% load i18n %}
 {% load tz %}
 
+{% block page_title %}
+  {{study_group.name}}
+{% endblock %}
+
+{% block page_meta %}
+  <meta name="description" content="Sign up for the learning circle {{ study_group.name }}">
+  <meta property="og:title" content="{{ study_group.name }}" />
+  {% if study_group.image %}
+    <meta property="og:image" content="{{ request.scheme }}://{{ request.get_host }}{{ study_group.image.url }}" />
+  {% else %}
+    <meta property="og:image" content="{{ request.scheme }}://{{ request.get_host }}{% static 'images/learning-circle-default.jpg' %}" />
+  {% endif %}
+  <meta property="og:type" content="website" />
```

Comment:
I wonder if we should use event as the type here? Would probably require other og: values

Reply:
I don't think event is one of the open graph types,  this is the list: https://ogp.me/#types